packages/web/src/components/edit-track/EditTrackForm/EditTrackForm.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { ThemeProvider } from '@audius/harmony'
+import { describe, it, expect } from 'vitest'
+
+// import { render, screen } from '@testing-library/react'
+import { render, screen } from '@audius/test'
+
+import { EditTrackForm } from './EditTrackForm'
+
+describe('EditTrackForm', () => {
+  it('renders correctly', () => {
+    render(
+      <ThemeProvider theme='day'>
```

Comment:
If using the custom render, why add the provider?

Reply:
yes great catch, forgot to remove :)